src/offline/Makefile

```diff
@@ -165,7 +168,25 @@ CINC = -I$(NCMOD)
 	$(FC) $(CFLAGS) $(CINC) -c $<
 
 # default target by convention is ``all''
-all : 
+all: serial
+
+serial: supr = -warn nostderrors -diag-disable 10145
```

Comment:
We probably shouldn't use `supr` here and instead append the flags to either `CFLAGS` or `LDFLAGS`.